docs/edge-authorizers/deployment-and-operation.mdx

```diff
@@ -75,9 +72,99 @@ docker run \
 -p 8282:8282 \
 -p 8383:8383 \
 -p 8484:8484 \
--v $PWD/certs:/certs:rw \
--v $PWD/cfg:/cfg:ro \
--v $PWD/eds:/db:rw \
+-p 9292:9292 \
+-v $HOME/.config/aserto/sidecar/certs:/certs:rw \
+-v $HOME/.config/aserto/sidecar/cfg:/cfg:ro \
+-v $HOME/.config/aserto/sidecar/eds:/db:rw \
 ghcr.io/aserto-dev/topaz:latest run \
---config-file /cfg/config.yaml
+--config-file /cfg/<POLICY_INSTANCE_NAME>.yaml
+```
+
+## Deploy as a Sidecar
+
+Deploying topaz as a sidecar next to your applications is the most efficient way to use the authorizer as it minimizes the authorization latency.
+
+In this section we can follow an example deployment on minikube based on a golang backend application and topaz authorizer as a sidecar.
```

Comment:
Golang
Topaz

Reply:
The language is usually spelled lowercase as "golang" similar to "node" or "python". Changed Topaz to "Edge Authorizer"